garrysmod/lua/vgui/dtree_node.lua

```diff
@@ -225,8 +225,9 @@ function PANEL:DoChildrenOrder()
 
 	if ( !IsValid( self.ChildNodes ) ) then return end
 
-	local last = table.Count( self.ChildNodes:GetChildren() )
-	for k, Child in pairs( self.ChildNodes:GetChildren() ) do
+	local children = self.ChildNodes:GetChildren()
+	local last = #children

```

Comment:
If you're gonna get the length of the table anyway, use a numeric-for loop

Reply:
You mean for i = 1, last do?

Reply:
I thought I could do that here, too. I think that's what you wanted. Can I mark it as resolved?

Reply:
Looks good.